lib/git-hooks.js

```diff
@@ -115,9 +115,10 @@ module.exports = {
      *
      * @param {String}   filename Path to git hook.
      * @param {String[]} [args]   Git hook arguments.
+     * @param {String}   input    Input string to be passed into each hook's standard input.
```

Comment:
Please do not align jsdoc :) 
It's harder to maintain (see code complete book).